k8s/openebs-monitoring/grafana-operator.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+spec:
+  type: NodePort
+  ports:
+  - port: 3000
+    targetPort: 3000
+    nodePort: 32525
```

Comment:
is this a standard port number? this will conflict with the number we used for -- jupyter for example

Reply:
Is there any doc issue that lists all the ports & service names used by Maya